static/gsAdmin/components/customers/customerStats.tsx

```diff
@@ -333,179 +333,176 @@ type Props = {
   projectId?: Project['id'];
 };
 
-export function CustomerStats({
-  orgSlug,
-  projectId,
-  dataType,
-  onDemandPeriodStart,
-  onDemandPeriodEnd,
-}: Props) {
-  const router = useRouter();
-
-  const dataDatetime = useMemo((): DateTimeObject => {
-    const {
-      start,
-      end,
-      utc: utcString,
-      statsPeriod,
-    } = normalizeDateTimeParams(router.location.query, {
-      allowEmptyPeriod: true,
-      allowAbsoluteDatetime: true,
-      allowAbsolutePageDatetime: true,
-    });
+export const CustomerStats = memo(
+  ({orgSlug, projectId, dataType, onDemandPeriodStart, onDemandPeriodEnd}: Props) => {
```

Comment:
this looks like a difficult diff to review but in reality, only this part was changed